android/src/main/kotlin/com/adyen/checkout/flutter/AdyenCheckoutPlugin.kt

```diff
@@ -102,9 +104,11 @@ class AdyenCheckoutPlugin : FlutterPlugin, ActivityAware {
                     }
 
                     dropInPlatformApi?.dropInAdvancedFlowCallback?.let {
-                        dropInPlatformApi?.dropInAdvancedFlowLauncher = DropIn.registerForDropInResult(
-                            fragmentActivity, it
-                        )
+                        dropInPlatformApi?.dropInAdvancedFlowLauncher =
```

Comment:
Nitpick, do you like this syntax?
```kotlin
dropInPlatformApi?.apply {
   dropInAdvancedFlowLauncher = DropIn.registerForDropInResult(
        fragmentActivity,
        dropInAdvancedFlowCallback,
    )
}
```

Reply:
That's a nice recommendation. Will use this syntax and change the code.